copilot-bootstrap.py

```diff
@@ -48,7 +48,7 @@
             Optional("backing-services"): [
                 {
                     "name": str,
-                    "type": lambda s: s in ("s3", "external-s3", "postgres", "redis", "opensearch",),
+                    "type": lambda s: s in ("s3", "s3-policy", "aurora-pg", "rds-pg", "redis", "opensearch",),
```

Comment:
To reduce potential for ambiguity/cognitive load, is there any reason we cannot use the full "postgres" word in these instead of "pg"?